### ISSUE TITLE
shadow-cljs -- Use of undeclared Var com.rpl.specter/java

### ISSUE THREAD

**Issue by knubie** [gnl/ghostwheel#15]
When requiring ghostwheel.core, shadow-cljs prints out the following warning after compilation.
```
------ WARNING #1 --------------------------------------------------------------
 File: com/rpl/specter.cljc:1272:19
--------------------------------------------------------------------------------
1269 |           ns (namespace structure)]
1270 |       (cond (keyword? structure) (keyword ns new-name)
1271 |             (symbol? structure) (symbol ns new-name)
1272 |             :else (i/throw-illegal "NAME can only be used on symbols or keywords - " structure)
-------------------------^------------------------------------------------------
 Use of undeclared Var com.rpl.specter/java
--------------------------------------------------------------------------------
1273 |             ))))
1274 |
1275 | (defnav ^{:doc "Navigates to the namespace portion of the keyword or symbol"}
1276 |   NAMESPACE
--------------------------------------------------------------------------------

------ WARNING #2 --------------------------------------------------------------
 File: com/rpl/specter.cljc:1285:19
--------------------------------------------------------------------------------
1282 |           new-ns (next-fn (namespace structure))]
1283 |       (cond (keyword? structure) (keyword new-ns name)
1284 |             (symbol? structure) (symbol new-ns name)
1285 |             :else (i/throw-illegal "NAMESPACE can only be used on symbols or keywords - " structure)
-------------------------^------------------------------------------------------
 Use of undeclared Var com.rpl.specter/java
--------------------------------------------------------------------------------
1286 |             ))))
1287 |
1288 | (defdynamicnav
1289 |   ^{:doc "Adds the result of running select with the given path on the
--------------------------------------------------------------------------------
```

I assume it has something to do with nathanmarz/specter#267.

**Comment by gnl** [comment]
I saw this recently after upgrading a few dependencies and yes, seems related. Not much to do on Ghostwheel's end here I think, but I'll leave the issue open until everything's sorted out with specter/shadow-cljs.

**Comment by gnl** [comment]
Fixed in fc0df5364d43f3514bbd8d34b12f454f912d18e8
